.github/workflows/ci.yml

```diff
@@ -51,10 +51,8 @@ jobs:
           python -m pip install --upgrade pip setuptools wheel
           python -m pip install --upgrade --upgrade-strategy eager -r requirements/dev.txt
           python -m pip install --upgrade --upgrade-strategy eager -e .
-          if [ "$RUNNER_OS" != "Windows" ] && ${{ matrix.python-version != '3.10.6' }}; then  # Skip Prophet tests on Windows as installation complex. Skip on Python 3.10 as not supported.
-            python -m pip install --upgrade --upgrade-strategy eager -e .[prophet]
-          fi
-          if [ "$RUNNER_OS" == "Linux" ]; then  # Currently, we only support KeOps on Linux.
+          python -m pip install --upgrade --upgrade-strategy eager -e .[prophet]
+          if [ "$RUNNER_OS" == "Linux"]; then  # Currently, we only support KeOps on Linux.
             python -m pip install --upgrade --upgrade-strategy eager -e .[keops]
           fi
           python -m pip install --upgrade --upgrade-strategy eager -e .[tensorflow,torch]
```

Comment:
nitpick: we could include `prophet` in with `[tensorflow,torch]` now? Or do you think it is best to separate backends from other optional deps?

Reply:
Makes sense, no particular reason to separate them now.